docs/ruleset_functions/is_baseline_system_6.md

```diff
@@ -0,0 +1,54 @@
+# is_baseline_system_6  
+
+**Description:** Get either Sys-6, Sys-6b, or Not_Sys_6 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 6 (Package VAV with PFP Boxes) or system 6b (system 6 with purchased heating).  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as either Sys-6, Sys-6b, or Not_Sys_6 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
```

Comment:
@claperle and @mkarpman why are we using system 7C (VAV with RH with PHW and PCHW) and not system 8C  VAV with PFPB with PCH and HW? i.e. why are we changing the terminal unit from PFP B to VAV with RH? for PCHW only, App G is quite clear that system 8 shall be used in pace of sys 6. However for both PHW and PCHW App G is not clear on what should be done instead of system 6.

Reply:
@supriyagoel I am not clear what you are refering to. I think I made the change you are referring to recently. I don't think any SYs-6s end up as Sys-7s. Can you please clarify?